recipe/meta.yaml

```diff
@@ -17,23 +18,23 @@ build:
     - vc10  # [win and py34]
     - vc14  # [win and py>34]
   skip: True  # [py<34 or win]
```

Comment:
I think this package is py3 only: https://github.com/python-rapidjson/python-rapidjson/blob/v0.0.10/setup.py#L21